examples/mcsh_example.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from ase import Atoms
+from ase.calculators.emt import EMT
+
+from amptorch.trainer import AtomsTrainer
+
+distances = np.linspace(2, 5, 10)
+images = []
+for dist in distances:
+    image = Atoms(
+        "CuCO",
+        [
+            (-dist * np.sin(0.65), dist * np.cos(0.65), 0),
+            (0, 0, 0),
+            (dist * np.sin(0.65), dist * np.cos(0.65), 0),
+        ],
+    )
+    image.set_cell([10, 10, 10])
+    image.wrap(pbc=True)
+    image.set_calculator(EMT())
+    images.append(image)
+
+
+sigmas = np.logspace(np.log10(0.05), np.log10(5.0), num=5)
+MCSHs = {   "MCSHs": {   
+                        "0": {"groups": [1], "sigmas": sigmas},
+                        "1": {"groups": [1], "sigmas": sigmas},
+                        "2": {"groups": [1,2], "sigmas": sigmas},
+                        "3": {"groups": [1,2,3], "sigmas": sigmas},
+                        "4": {"groups": [1,2,3,4], "sigmas": sigmas},
+                        "5": {"groups": [1,2,3,4,5], "sigmas": sigmas},
+                        "6": {"groups": [1,2,3,4,5,6,7], "sigmas": sigmas},
+                        #"7": {"groups": [1,2,3,4,5,6,7,8], "sigmas": sigmas},
+                        #"8": {"groups": [1,2,3,4,5,6,7,8,9,10], "sigmas": sigmas},
+                        #"9": {"groups": [1,2,3,4,5,6,7,8,9,10,11,12], "sigmas": sigmas}
+                  },
+            "atom_gaussians": {
+            "C": "./MCSH_potential/C_coredensity_5.g",
+			"H": "./MCSH_potential/H_totaldensity_3.g",
```

Comment:
Are these files required for MCSH? If so, we should probably include them for MCSH use. If they're above >10MB though we can make it a separate download instead though.

Reply:
yes, you can think them as parameter files for different atom types, I can include some example ones in the examples directory